### PR TITLE
add AngularCamp

### PR DESCRIPTION
**Pull request by davidpich**

### CODE REVIEW

File: conferences/2018/javascript.json [hunk 1]
```diff
@@ -379,14 +379,21 @@
...
+    "country": "Spain"
+  },
   {
     "name": "Node Summit",
     "url": "http://www.nodesummit.com/",
     "startDate": "2018-07-23",
     "endDate": "2018-07-25",
     "city": "San Francisco, CA",
     "country": "U.S.A.",
-    "twitter": "@NodeSummit"
```
Comment (rossjs) [inline conferences/2018/javascript.json]: Did you accidentally delete this?

**Comment by nimzco** [comment]
I've just added it on master, thanks! 👍

File: conferences/2018/javascript.json (hunk 1 above)
Comment (davidpich) [reply conferences/2018/javascript.json]: Sorry, It was my fault, I accidentally did it 🙏

**Comment by davidpich** [comment]
Thank you, and sorry for the inconvenience, my fault to don't review it well, it was not my intention to delete anything 🙏

**Comment by rossjs** [comment]
Don't sweat it! Thanks for contributing!

**Comment by davidpich** [comment]
First at all thanks and happy new beginning of 2018 to all! Now I see I forgot to add the twitter handle @AngularCamp, Can I do another PR?

**Comment by prigara** [comment]
Hi @davidpich! Thanks!
I've added it myself :)

**Comment by davidpich** [comment]
Thank you!

**Pull request closed without merging**